vite.config.js

```diff
@@ -0,0 +1,60 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue2'
+import EnvironmentPlugin from 'vite-plugin-environment'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('fs')
+const packageJson = fs.readFileSync('./package.json')
+const appName = JSON.parse(packageJson).appName
+const appVersion = JSON.parse(packageJson).version
+const sbcName = JSON.parse(packageJson).sbcName
+const sbcVersion = JSON.parse(packageJson).dependencies['sbc-common-components']
+const aboutText1 = (appName && appVersion) ? `${appName} v${appVersion}` : ''
+const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
+
+// https://vitejs.dev/config/
+export default defineConfig(() => {
+  return {
+    define: {
+      'import.meta.env.ABOUT_TEXT':
+        (aboutText1 && aboutText2) ? `"${aboutText1}<br>${aboutText2}"`
+          : aboutText1 ? `"${aboutText1}"`
+            : aboutText2 ? `"${aboutText2}"`
+              : ''
+    },
+    envPrefix: 'VUE_APP_', // Need to remove this after fixing vaults. Use import.meta.env with VUE_APP.
+    plugins: [
+      vue(),
+      EnvironmentPlugin({
+        BUILD: 'web' // Fix for Vuelidate, allows process.env with Vite.
+      })
+    ],
+    resolve: {
+      alias: {
+        '@': path.resolve(__dirname, './src')
+      },
+      extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json', '.vue']
+    },
+    server: {
+      host: true,
+      port: 8080
+    },
+    test: {
+      // simulate DOM with happy-dom
+      environment: 'happy-dom',
+      // enable jest-like global test APIs
+      globals: true,
+      setupFiles: ['./tests/setup.ts'],
+      // enable threads to speed up test running
+      threads: true,
+      // hide Vue Devtools message
+      onConsoleLog: function (log) {
+        if (log.includes('Download the Vue Devtools extension')) {
```

Comment:
Weird that there's isn't a property override to hide this message, along the lines of this:
https://github.com/bcgov/business-filings-ui/blob/35248572ab7c39a5717ae7378c25ef7812ff89eb/src/main.ts#L24

Reply:
Agreed.

Reply:
So there isn't a property  you can set?

Reply:
I've made some research online. I can't find anything or anyone mentioning if there's a property override for this. I also took a look at the documentation, I couldn't find a property like the one you mentioned. I'm assuming this is the only way to hide the message here?

Reply:
What about setting `Vue.config.devtools=false`? [ref](https://stackoverflow.com/a/54947933/8679162)

Or, fuggedaboutit.

Reply:
Actually, I've found a super cool way to do it 😁 Please check the latest commit Sev!

Reply:
It might be a good idea to make this change for both Edit and Create too 👍

Reply:
Let's give this 5 more minutes of thought and then I vote to fuggedaboutit.

Reply:
Are you reverting the vitest config change?

Reply:
I thought we'll hide it only with test coverage. I'll revert the change Sev, on it!

Reply:
If the silent option hides unit test errors and console statements then we don't want that option when running the tests manually.

For the test coverage option, our goal is to see coverage info, not errors, so the silent option makes sense (assuming the coverage info is not silenced as well).

Reply:
Got it. Sounds good Sev 👍